webpack/envs/clientNovoDevelopmentConfig.js

```diff
@@ -200,13 +213,8 @@ export const clientNovoDevelopmentConfig = {
     }),
     new HtmlWebpackPlugin({
       filename: path.resolve(basePath, "public", "index.ejs"),
-      template: `!!raw-loader!${path.resolve(
-        basePath,
-        "src",
-        "novo",
-        "src",
-        "index.ejs"
-      )}`,
+      inject: false,
```

Comment:
(thought) The PR title says "inject automatically", but this is set to `false`. Looking at [the docs](https://www.npmjs.com/package/html-webpack-plugin#hello-world-5) this seems to determine automatic injection.

Reply:
Yes, `inject: false` disables automatic injection before the `</body>` tag and allows the injection point to be manually determined using a template variable instead.